usaspending_api/etl/es_etl_helpers.py

```diff
@@ -384,12 +462,19 @@ def create_aliases(client, index, silent=False):
     # If the alias is on multiple indexes, the loads will fail!
     printf(
         {
-            "msg": "Putting alias '{}' on {}".format(settings.ES_TRANSACTIONS_WRITE_ALIAS, index),
+            "msg": "Putting alias '{}' on {}".format(
+                settings.ES_AWARDS_WRITE_ALIAS if load_type == "awards" else settings.ES_TRANSACTIONS_WRITE_ALIAS, index
+            ),
             "job": None,
             "f": "ES Alias Put",
         }
     )
-    put_alias(client, index, settings.ES_TRANSACTIONS_WRITE_ALIAS, {})
+    put_alias(
+        client,
+        index,
+        settings.ES_AWARDS_WRITE_ALIAS if load_type == "awards" else settings.ES_TRANSACTIONS_WRITE_ALIAS,
```

Comment:
Is this was stored as a variable, then the function calls wouldn't explode across many lines